Zend/zend_portability.h

```diff
@@ -313,6 +313,18 @@ char *alloca();
 # define ZEND_FASTCALL
 #endif
 
+#if __has_attribute(preserve_none) && !defined(__SANITIZE_ADDRESS__)
```

Comment:
There is an incompatibility between `preserve_none` and ASAN, which crashes Clang. I will report the issue.